db/models.py

```diff
@@ -50,3 +51,42 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to="db.User", on_delete=models.CASCADE, related_name="orders"
```

Comment:
Ensure that the `User` model exists in the `db` module. If it doesn't, you need to create it or import it from the appropriate module.